### ISSUE TITLE
Update requirements.txt

### ISSUE THREAD

**Issue by anirudhpnbb** [anirudhpnbb/Pyostie#60]
**Describe the bug**
Few libraries not included in the requirements.txt file.

**Comment by massone99** [comment]
Hi, I can do it if you like

**Comment by anirudhpnbb** [comment]
Please go ahead. Create a pull request and work on it. Commit and push to a new branch.

Thanks

**Comment by massone99** [comment]
![immagine](https://github.com/user-attachments/assets/7b4083d8-eeaf-4d4c-85f7-1116700195f8)
I used https://github.com/bndr/pipreqs to retrieve the requirements. It seems the majority of the requirements inside the previous version are no where to be found. I guess you added them but never used them. They are less than before but I checked and all the libraries in the current requirements.txt are the only libraries **currently used**. Does this make sens to you?

**Comment by anirudhpnbb** [comment]
Yeah this sounds good. Can you create a PR so that I can test and merge?
@massone99
